### PR TITLE
Feature/types

### PR DESCRIPTION
**Pull request by DovydasNavickas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,9 @@
     "license": "AGPL",
     "dependencies": {
         "@types/fs-extra": "^4.0.2",
-        "read-package-json": "^2.0.10",
         "fs-extra": "^4.0.2",
+        "path-is-inside": "^1.0.2",
```
Comment (MartynasZilinskas) [inline package.json]: We should remove this dependency.

File: src/api-helpers.ts [hunk 2]
```diff
@@ -108,6 +115,42 @@ export namespace ApiHelpers {
         return apiItem;
     }
 
+    export function PathIsInside(thePath: string, potentialParent: string): boolean {
```
Comment (MartynasZilinskas) [inline src/api-helpers.ts]: I think we could move this to `utils` in a separate file?

Comment (DovydasNavickas) [reply src/api-helpers.ts]: FIxed.

File: package.json (hunk 1 above)
Comment (DovydasNavickas) [reply package.json]: Fixed.

**Comment by DovydasNavickas** [comment]
Fixed.

File: src/utils/path-is-inside.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    if (process.platform === "win32") {
+        thePath = thePath.toLowerCase();
+        potentialParent = potentialParent.toLowerCase();
+    }
+
+    thePath = path.normalize(thePath);
+    potentialParent = path.normalize(potentialParent);
+
+    return thePath.lastIndexOf(potentialParent, 0) === 0 &&
+        (
```
Comment (GiedriusGrabauskas) [inline src/utils/path-is-inside.ts]:
```ts
... && ( 
...
)
```

Comment (MartynasZilinskas) [reply src/utils/path-is-inside.ts]: Fixed.

**Pull request merged**